src/main/java/org/sonarsource/sonarlint/core/container/rule/RuleFinderCompatibility.java

```diff
@@ -111,12 +112,14 @@ private static Rule toRule(@Nullable RulesDefinition.Rule ar) {
     return ar == null ? null : toRuleNotNull(ar);
   }
 
-  private static Rule toRuleNotNull(RulesDefinition.Rule ar) {
-    Rule rule = Rule.create(ar.repository().key(), ar.key())
-      .setName(ar.name())
-      .setLanguage(ar.repository().language())
-      .setConfigKey(ar.internalKey());
-    for (Param param : ar.params()) {
+  private static Rule toRuleNotNull(RulesDefinition.Rule ruleDef) {
+    Rule rule = Rule.create(ruleDef.repository().key(), ruleDef.key())
+      .setName(ruleDef.name())
+      .setSeverity(RulePriority.valueOf(ruleDef.severity()))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "RulePriority"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)